test/ripper/test_ripper.rb

```diff
@@ -142,7 +142,7 @@ def test_dedent_string
   end
 
   def test_no_memory_leak
-    assert_no_memory_leak(%w(-rripper), "", "#{<<~'end;'}", rss: true)
+    assert_no_memory_leak(%w(-rripper), "", "#{<<~'end;'}", rss: true, timeout: 60)
```

Comment:
This test consumes 22 secs.
Repeating 10M times is needed?

Reply:
I changed it to 2M times. At 1M times, the test doesn't work on my M1 MacBook Pro (but works on my x86 Linux machine). 2M times is the minimum for my M1.